src/components/passphrase/passphrase.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import Button from 'react-toolbox/lib/button';
+import Input from 'react-toolbox/lib/input';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import styles from './passphrase.css';
+import InfoParagraph from '../infoParagraph';
+import PassphraseGenerator from './passphraseGenerator';
+import PassphraseConfirmator from './passphraseConfirmator';
+import steps from './steps';
+
+class Passphrase extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      steps: steps(this),
+      currentStep: 'info',
+      answer: '',
+    };
+  }
+
+  changeHandler(name, value) {
+    this.setState({ [name]: value });
+  }
+
+  render() {
+    const templates = {};
```

Comment:
I've just moved configs there, not templates. templates are big enough to stuff whichever file we put them in.